app/Platform/Controllers/GameHashController.php

```diff
@@ -27,10 +28,13 @@ public function index(Request $request, Game $game): InertiaResponse
         $this->authorize('viewAny', $this->resourceClass());
 
         $gameData = GameData::fromGame($game)->include('badgeUrl', 'forumTopicId', 'system');
-        $hashes = GameHashData::fromCollection($game->hashes);
+        $hashes = GameHashData::fromCollection($game->hashes->where('compatibility', GameHashCompatibility::Compatible));
+        $incompatibleHashes = GameHashData::fromCollection($game->hashes->where('compatibility', GameHashCompatibility::Incompatible));
+        $untestedHashes = GameHashData::fromCollection($game->hashes->where('compatibility', GameHashCompatibility::Untested));
+        $patchRequiredHashes = GameHashData::fromCollection($game->hashes->where('compatibility', GameHashCompatibility::PatchRequired));
```

Comment:
May be good to think about using scopes for these if we find ourselves reaching for compatibility values often elsewhere.

Reply:
I added a `compatibleHashes` property on game, and a `scopeCompatible` on `GameHash`. I couldn't figure out how to use the scope on the `game->hashes` collection.